lib/animina/user_email.ex

```diff
@@ -29,20 +27,19 @@ defmodule Animina.UserEmail do
     "Hi #{user.name}!\n"
   end
 
-  defp construct_email_body do
-    ~S"""
-
-    Your new ANIMINA https://animina.de account has been created with this
+  defp construct_email_body(user) do
+    "Your new ANIMINA https://animina.de account has been created with this
     email address.
 
-    Please confirm the email address by clicking the monstrosity of a link
-    below or do nothing in case you didn't create the account.
+    Below is your super secret 6 digit pin that you can use to verify your account once you
+    visit  https://animina.de/my/email-validation , you have only 3 attemps so make sure
```

Comment:
A user might create a new account on his desktop but reads this email on his mobile phone. Clicking on the URL in the mobile phone will get him nowhere.

Better text:
Please use this 6 digit PIN to verify your new ANIMINA account:

Reply:
This has been done , thanks